services/ux-backend/handlers/bucket/handler.go

```diff
@@ -0,0 +1,86 @@
+package bucket
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+
+	nbv1alpha1 "github.com/noobaa/noobaa-operator/v5/pkg/apis/noobaa/v1alpha1"
```

Comment:
```suggestion
	nbv1a1 "github.com/noobaa/noobaa-operator/v5/pkg/apis/noobaa/v1alpha1"
```
nit, change if you are fine & refreshing the PR.